visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -413,9 +400,26 @@ export class CodeMapMouseEventService implements ViewCubeEventPropagationSubscri
 		return this.mouse.x !== x || this.mouse.y !== y
 	}
 
+	private isGrabbingOrMoving() {
+		return this.isGrabbing || this.isMoving
+	}
+
 	private hoverBuilding(hoveredBuilding: CodeMapBuilding) {
-		if (hoveredBuilding) {
-			this.hoverBuildingAndChildren(hoveredBuilding)
+		CodeMapMouseEventService.changeCursorIndicator(CursorType.Pointer)
```

Comment:
Why we don't need isGrabbingOrMoving() here?

Reply:
There is only a single [call site](https://github.com/MaibornWolff/codecharta/pull/2511/files/a72c4acaf7286c8f71d0a983a06d2411b0b6d91c#diff-c80ef911d9938116f514f83e9a6652169f64075c015fefcee96ea555bf1a54a1L140) for this function at the moment and it has some further logic that is not required in case `isGrabbingOrMoving()` returns true. Thus, it already checks that in that function.

https://github.com/MaibornWolff/codecharta/pull/2511/files/a72c4acaf7286c8f71d0a983a06d2411b0b6d91c#diff-c80ef911d9938116f514f83e9a6652169f64075c015fefcee96ea555bf1a54a1R135-R137